viewing_party/party.py

```diff
@@ -0,0 +1,128 @@
+def create_movie(title, genre, rating):
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    if new_movie["title"] == None:
+        return None
+    elif new_movie["genre"] == None:
+        return None
+    elif new_movie["rating"] == None:
+        return None
+    return new_movie
+
+
+def add_to_watched(user_data,movie):
+    # user_data = {"watched":[]}
+    # movie = {"title": "Title A","genre":"Horror","rating":3.5}
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data,movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data,title):
+    for item in user_data["watchlist"]:
+        if item["title"] == title:
+            user_data["watchlist"].remove(item)
+            user_data["watched"].append(item)
+    return user_data
+
+
+def get_watched_avg_rating(user_data):
+    #look at the value that is a list
+    #looping thru the list
+    # get the value of  "rating"
+    #user_data = {"watched":[]}
+    moviesList = user_data["watched"]
+    if moviesList == []:
+            return 0.0
+    sum = 0
+    for item in moviesList:
+        # print(f"Looking at {item=}")
+        sum += item["rating"]
+        
+    avg = sum/len(moviesList)
+    return avg
+
+def get_most_watched_genre(user_data):
+    # user_data = {"watched":[{"genre":"something"},
+    # {"genre":"romantic"},{"genre":"something"}]
+    genre_count = {}
+    max_watched = None
+    
+    for item in user_data["watched"]:
+        if (item["genre"] in genre_count):
+            genre_count[item["genre"]] += 1
+        else:
+            genre_count[item["genre"]] = 1
+        for key,value in genre_count.items():
+            max_watched = max(genre_count, key =genre_count.get)
+            # genre_max_list.append(value)
+
+    return max_watched
+
+
+def get_unique_watched(user_data):
```

Comment:
👍 This works and is fine, but in the future you may want to consider using a `set` or dictionary here as the `if friend_watched in user_data["watched"]:` statement basically is a loop trying to find `friend_watched` in the given list.